src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -83,10 +102,10 @@
       + "initial connection to discover the full cluster membership (which may change "
       + "dynamically), "
       + "this list need not contain the full set of servers (you may want more than one, though, "
-      + "in case a server is down).";
+      + "in case a server is down). While it is required to configure either of this or "
```

Comment:
I still didn't want to use depreciated because we would need to depreciated V1 API's first